CONTRIBUTING.md

```diff
@@ -1,193 +1,169 @@
 
-```md
-<meta http-equiv="refresh" content="0; url=https://cp-algorithms.com/<new section>/<new name>.html">
-# <Article name>
-Article was moved (renamed). <a href="<new section>/<new name>.html">new URL</a>.
+```markdown
+---
+title: Alternative HTML Title
+---
+# Proof of $a^2 + b^2 = c^2$
 ```
 
-### Linking to a section with anchors
+### Handling Redirects
 
-Also it's kind of problematic when renaming a section of an article.
-The section title is used for linking.
-E.g. a section on the page `article.md` with
+If you move or rename an article, make sure to set up a redirect. A redirect file should look like this:
 
 ```md
-## Some title
+<meta http-equiv="refresh" content="0; url=https://cp-algorithms.com/<new-section>/<new-article>.html">
+# Article Name
+This article has been moved to a [new location](<new-section>/<new-article>.html).
 ```
 
-can be linked to with `/article.html#some-title`.
-If the title is changed, the link doesn't work any more, and this breaks links from other articles or other websites.
+### Maintaining Anchor Links
 
-If you rename an article, insert an anchor so that the old link still works:
+If you rename a section, the link to that section might break. To avoid this, add an anchor manually:
 
 ```html
-<div id="some-title"></div>
+<div id="old-section-title"></div>
 ```
 
-### Tags
+This will allow existing links to continue working even after the section title is changed.
 
-To distinguish original and translatory articles, they should be marked with corresponding tags. For original articles, it's
+### Article Tags
 
-```md
----
-tags:
-    - Original
----
-```
+We use tags to differentiate between original content and translated articles. Add the appropriate tag at the top of your article:
 
-And for translated articles, it's
+- **For original articles**:
 
-```md
----
-tags:
-    - Translated
-e_maxx_link: ...
----
-```
-
-Here, instead of `...` one should place the last part of the link to the original article. E.g. for [Euler function article](http://e-maxx.ru/algo/euler_function) it should be
+    ```md
+    ---
+    tags:
+        - Original
+    ---
+    ```
 
+- **For translated articles**:
 
-```md
----
-tags:
-    - Translated
-e_maxx_link: euler_function
----
-```
+    ```md
+    ---
+    tags:
+        - Translated
+    e_maxx_link: <original-link>
+    ---
+    ```
 
+    Replace `<original-link>` with the URL to the source article.
 
-## Some conventions
+## Conventions
 
-* We have agreed as of issue [#83](https://github.com/cp-algorithms/cp-algorithms/issues/83) to express binomial coefficients with `\binom{n}{k}` instead of `C_n^k`. The first one renders as $\binom{n}{k}$ and is a more universal convention. The second would render as $C_n^k$.
+We follow certain conventions across the project. For example, we agreed to use the `\binom{n}{k}` notation for binomial coefficients instead of `C_n^k` as outlined in [issue #83](https://github.com/cp-algorithms/cp-algorithms/issues/83).
```

Comment:
Why did you resolve this? Nothing changed here...

Reply:
```suggestion
We follow certain conventions across the project. For example, we agreed to use the `\binom{n}{k}` notation for binomial coefficients instead of `C_n^k` as outlined in [issue #83](https://github.com/cp-algorithms/cp-algorithms/issues/83). The first one renders as $\binom{n}{k}$ and is a more universal convention. The second would render as $C_n^k$.
```